src/com/carolinarollergirls/scoreboard/defaults/DefaultClockModel.java

```diff
@@ -207,8 +209,13 @@ else if (ms > maximumTime && ms - maximumTime > 500)
 		else
 			return ms;
 	}
-	protected boolean checkStop() {
-		return (getTime() == (isCountDirectionDown() ? getMinimumTime() : getMaximumTime()));
+	protected boolean isDisplayChange(long current, long last) {
+		//on count down clocks are rounded down for display, on count up they are rounded down.
```

Comment:
This comment reads oddly, and I don't understand why the directions have different logic.